tutorial/installing_the_diagnostics_tools.md

```diff
@@ -0,0 +1,48 @@
+# Installing the diagnostics tools
+
+Depending on the diagnostics scenario you will use one or more of the tools below to get to root cause. By default, these tools are installed to ~/.dotnet/tools. 
+
+### dotnet-counters
+In the .NET full/Windows world, we have a myriad of performance counters that can be used to triage and diagnose production issues. For .Net core we have a similar and cross platform story centered around a tool called dotnet-counters. To install the tool, run the following command:
+
+> ```bash
+> dotnet tool install --global dotnet-counters --version 1.0.3-preview5.19228.1 --add-source https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json`
```

Comment:
We've published this packages to NuGet now so you no longer need to use --add-source. I think the latest version install looks like this:

`dotnet tool install --global dotnet-counters --version 1.0.3-preview5.19251.2`